### ISSUE TITLE
Cube Creation in Abaqus

### ISSUE THREAD

**Issue by aspgaonk** [Mote3D/Mote3D_toolbox#6]
Hey,
So after importing the respective .py script into Abaqus and running the script, the particles are generated but not merged to one full part/cube, any clues as to how to do it? Thank you.

**Comment by aspgaonk** [comment]
> 
> 
> Hey, After clicking on "Input Script" I cannot seem to find the location of where the file "Abq_voxel_mesh.py" is saved, I'd grateful if anyone can help me with this. Thank you.
> Regards,
> Ashwin

SO,

**Comment by aspgaonk** [comment]
SO, my directory wasn't changing for some reason, and files were saved in the "users/name" directory, which I realised later. THa

**Comment by aspgaonk** [comment]
Also, what is the dimension of the domain, its default values is 5.

**Comment by Mote3D** [comment]
Mote3D generates a random particle distribution. If you import the geometry into Abaqus, Boolean operations (merge, cut, fuse etc.) can be used to combine the particles with other parts (created directly in Abaqus) to create multiphase microstructures.

If you would like to create a cubic domain containing particles, you need to import the particle distribution first, then create a solid cube in Abaqus, then subtract the particle domain from the cube (Boolean cut) in the Abaqus assembly to create a "matrix domain", and then merge the particles with this "matrix domain".

The dimensions of the domain are arbitrary.

**Comment by Mote3D** [comment]
By the way, there's a video on youtube that describes the process:
[https://www.youtube.com/watch?v=Exvxa4xouBs](https://www.youtube.com/watch?v=Exvxa4xouBs)

**Comment by aspgaonk** [comment]
Thank you! That videos definitely helps.

Regards,
Ashwin Gaonkar

Sent from my iPhone

On Aug 11, 2020, at 09:52, Mote3D <notifications@github.com> wrote:

﻿
This message was sent from a non-IU address. Please exercise caution when clicking links or opening attachments from external sources.


By the way, there's a video on youtube that describes the process:
https://www.youtube.com/watch?v=Exvxa4xouBs

**Comment by tarkesdora** [comment]
I tried to create a particulate composite microstructure with a variance of diameter as zero (eg picture as shown in the first page github  MOTE3D) by merging with a cube. I face merge error always. I tried with vol fraction of 0.1 and 0.25. Any help or directions I should try in this regard would be helpful. Generating a porous microstructure is very easy by using cut option of ABAQUS in the assembly module.
